source/manual/readmes.html.md

```diff
@@ -7,105 +7,73 @@ type: learn
 parent: "/manual.html"
 ---
 
-This is a guide to writing and maintaining README documents for GOV.UK's public repositories.
+This is a guide to writing and maintaining README documents for GOV.UK's public repositories. It's based on [guidance we have in the GDS Way for READMEs](https://gds-way.cloudapps.digital/manuals/readme-guidance.html#writing-readmes).
 
-## Who are READMEs for (aka what's the user need)?
+READMEs are for a technical audience. This could be a new starter or an existing developer. Using the following template will help to ensure each README is consistent, correct, short and actionable.
 
-- New starters to GDS
-- People joining a new team
-- People interested in our work
-- Developers on other teams
+Unless it helps someone to get started, any other content should be a separate file in the `docs/` directory for a repository. This also makes the content easier to discover in external search.
 
 ## Template for new READMEs
 
 ```markdown
 # App name
 
-One paragraph description and purpose.
+One or more paragraphs describing the app:
 
-## Screenshots (if there's a client-facing aspect of it)
+- What it's used for.
+- Any unusual dependencies it has.
 
-## Live examples (if available)
+## Live examples
 
-- [gov.uk/thing](https://www.gov.uk/thing)
+(Only applies to frontend apps.)
+
+A list of links to example pages rendered by the app.
 
 ## Nomenclature
```

Comment:
Can we say "Defined terms" or something more plain English than "Nomenclature"?

Reply:
How about just "Definitions"?

("Defined terms" leads me to question where the "Undefined terms" are listed...)

Reply:
"Definitions" works for me

Reply:
Cool. Are you comfortable if we defer changing this to a follow-up PR? At the moment all our READMEs use the current wording, so ideally we should change them so the template is consistent with reality.

There's already a task on [the follow-up card](https://trello.com/c/VU3R1B8X/240-make-govuk-readmes-consistent-with-the-gds-way) to rename "Technical documentation" to "Getting started", which will involve touching every app, so we could do it as part of that.

Reply:
sure

Reply:
OK. I've made a note on the follow-up card, and will leave this comment unresolved.